HowToRPC.md

```diff
@@ -0,0 +1,64 @@
+# How to Call Google APIs: RPC Edition
+
+_For many developers, the easiest way to call a Google API is with one of our client libraries. But occasionally someone may prefer to make API calls directly - perhaps from a language or environment that we don’t support or using a different networking library or tool. Here we’ll show you how to do it._
+
+This page focuses on calling Google APIs directly using their underlying RPC interfaces. Most of these APIs are also available as REST services. For that, see [How to Call Google APIs, REST Edition](/HowToREST).
+
```

Comment:
A paragraph saying "What is RPC?" would be valuable here.

Reply:
Adding links out to https://en.wikipedia.org/wiki/Remote_procedure_call for now.